services/basic_types.proto

```diff
@@ -1179,7 +1179,9 @@ message NodeAddress {
     bytes memo = 3 [deprecated=true];
 
     /**
-     * The node's hex-encoded X509 RSA public key
+     * Hex encoding of the node's X509 RSA public key used to sign stream files (e.g., record 
+     * stream files). That is, this field is a string of hexadecimal characters which, translated 
```

Comment:
Remove `That is, ` or use `Specifically, ` as in the other change.

Reply:
Used "Specifically, "...